.github/workflows/deployment.yml

```diff
@@ -0,0 +1,27 @@
+name: Deployment
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy_docs:
+    runs-on: ubuntu-latest
+    container:
+      image: crystallang/crystal:latest-alpine
+    steps:
+      - name: Install Build Dependencies
+        run: apk add --update rsync
+      - uses: actions/checkout@v2
+      - name: Install Dependencies
+        run: shards install --production
+      - name: Build
+        run:  crystal docs lib/amq-protocol/src/amq-protocol.cr src/amqp-client.cr
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@3.5.1
```

Comment:
Can we use the latest version, 3.6.1? (https://github.com/JamesIves/github-pages-deploy-action/releases)

Reply:
Ah jeez, was on the 2nd page :see_no_evil:.